dev/io.openliberty.mcp/src/io/openliberty/mcp/annotations/OutputSchemaGenerator.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
```

Comment:
Should these also have EPL license?

Reply:
I didn't think so but I'm happy to be told what it should say. Who would I need to ask about this?